src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { Colors } from '../../essentials';
+import { Box } from '../Box/Box';
+import Compact from './components/Compact';
+import DefaultPanel from './components/Default';
+
+interface AccordionProps {
+    title?: string;
+    description?: string;
+    info?: string;
+    buttonLabel?: string;
+    variant?: 'compact' | 'default';
+    expanded?: boolean;
+    children: ReactNode;
+}
+
+const HorizontalDivider = styled(Box)`
+    border: 0;
+    border-top: solid 1px ${Colors.AUTHENTIC_BLUE_200};
+`;
+
+const HorizontalDividerTop = HorizontalDivider;
+
+const HorizontalDividerBottom = styled(HorizontalDivider)`
+    display: none;
+`;
+
+const RenderedSection = styled(Box)`
+    :last-child ${HorizontalDividerBottom} {
+        display: inherit;
+    }
+`;
+
+const Accordion = ({ title, description, info, buttonLabel, variant, expanded, children }: AccordionProps) => (
```

Comment:
One of our requirements on this Accordion is to change the text when it's either opened or closed. Maybe we could let the parent component take care of the state in this case?